[move reference] Add references to Sui (#16785)
MystenLabs/sui: external-crates/move/documentation/book/src/functions.md

fun just_unit() { }
```

> As mentioned in the [tuples section](./primitive-types/tuples.md), these tuple "values" are virtual
> and do not exist at runtime. So for a function that returns unit `()`, it will not be returning any
> value at all during execution.

Reviewer: ```suggestion
As mentioned in the [tuples section](./primitive-types/tuples.md), these tuple "values" 
do not exist as runtime values. This means that a function that returns unit `()` not be returning any
value during execution.
```

Author: ```suggestion
As mentioned in the [tuples section](./primitive-types/tuples.md), these tuple "values" 
do not exist as runtime values. This means that a function that returns unit `()` does not return any value during execution.
```
